### PR TITLE
Fix for getStroke issue by checking

### PR DESCRIPTION
**Pull request by schagani**
style variable if it is array or object

### CODE REVIEW

**Review by adube — CHANGES_REQUESTED** [review]
Thanks for the contribution

Run `npm run lint` and fix the issues raised there.

File: src/olgm/gm.js [hunk 1]
```diff
@@ -226,7 +226,14 @@ export function createStyleInternal(style, mapIconOptions, opt_index) {
   // strokeColor
   // strokeOpacity
   // strokeWeight
-  const stroke = style.getStroke();
+   // Checking style variable if it is array or object
+  let stroke;
+  if(style instanceof Array){
```
Comment (adube) [inline src/olgm/gm.js]: Use `Array.isArray` instead.

Do so for the equivalent code below.

**Comment by schagani** [comment]
Changes done. Please review.

File: src/olgm/gm.js [hunk 2]
```diff
@@ -226,7 +226,14 @@ export function createStyleInternal(style, mapIconOptions, opt_index) {
   // strokeColor
   // strokeOpacity
   // strokeWeight
-  const stroke = style.getStroke();
+  // Checking style variable if it is array or object
+  let stroke;
+  if (style instanceof Array) {
```
Comment (adube) [inline src/olgm/gm.js:231]: Replace by Array.isArray

File: src/olgm/gm.js [hunk 3]
```diff
@@ -245,7 +252,13 @@ export function createStyleInternal(style, mapIconOptions, opt_index) {
 
   // fillColor
   // fillOpacity
-  const fill = style.getFill();
+  let fill;
+  if (style instanceof Array) {
```
Comment (adube) [inline src/olgm/gm.js:256]: Replace by Array.isArray

File: src/olgm/gm.js [hunk 4]
```diff
@@ -257,7 +270,13 @@ export function createStyleInternal(style, mapIconOptions, opt_index) {
     }
   }
 
-  const image = style.getImage();
+  // Checking style variable if it is array or object
+  let image;
+  if (style instanceof Array) {
```
Comment (adube) [inline src/olgm/gm.js:275]: Replace by Array.isArray

File: src/olgm/herald/Feature.js [hunk 5]
```diff
@@ -229,19 +231,34 @@ class FeatureHerald extends Herald {
     const style = getStyleOf(this.feature_);
     if (style) {
       const latLng = createLatLng(getCenterOf(this.getGeometry_()));
-
-      const zIndex = style.getZIndex();
+      // Checking style variable if it is array or object
+      let zIndex;
+      if (style instanceof Array) {
```
Comment (adube) [inline src/olgm/herald/Feature.js:236]: Replace by Array.isArray

File: src/olgm/herald/Feature.js [hunk 6]
```diff
@@ -229,19 +231,34 @@ class FeatureHerald extends Herald {
...
+        zIndex = style[0].getZIndex();
+      } else {
+        zIndex = style.getZIndex();
+      }
       const index = zIndex !== undefined ? zIndex : this.index_;
-
-      const image = style.getImage();
+      // Checking style variable if it is array or object
+      let image;
+      if (style instanceof Array) {
```
Comment (adube) [inline src/olgm/herald/Feature.js:244]: Replace by Array.isArray

File: src/olgm/herald/Feature.js [hunk 7]
```diff
@@ -229,19 +231,34 @@ class FeatureHerald extends Herald {
...
         this.marker_ = createMapIcon(image, latLng, index, this.feature_.get('olgm_pane'));
         if (this.visible_) {
           this.marker_.setMap(this.gmap);
         }
       }
-
-      const text = style.getText();
+      // Checking style variable if it is array or object
+      let text;
+      if (style instanceof Array) {
```
Comment (adube) [inline src/olgm/herald/Feature.js:257]: Replace by Array.isArray

**Review by adube — CHANGES_REQUESTED** [review]
Remove all changes from package-lock.json.

See inline comments below too.

**Comment by adube** [comment]
The OLGM library is now deprecated.  See: https://github.com/mapgears/ol3-google-maps/pull/352

**Pull request closed without merging**